README.md

```diff
@@ -4,13 +4,50 @@ Please edit this README.md file with your answers to the worksheet questions.
 
 ## Question 1
```

Comment:
Truth tables look correct, but would be easier to read with column headings and correct markdown formatting etc.